Sources/HipMobileUI/Pages/LoadingPage.xaml

```diff
@@ -33,32 +33,33 @@
                         <ColumnDefinition Width="*" />
                     </Grid.ColumnDefinitions>
                     <Grid.RowDefinitions>
-                        <RowDefinition Height="0.4*" />
-                        <RowDefinition Height="0.5*" />
+                        <RowDefinition Height="0.3*" />
+                        <RowDefinition Height="0.6*" />
                         <RowDefinition Height="0.1*" />
                     </Grid.RowDefinitions>
-                    <StackLayout Grid.Row="0" Orientation="Vertical" Spacing="20" HorizontalOptions="CenterAndExpand"
+                    
+                    <StackLayout Grid.Row="0" Orientation="Vertical" Spacing="0" HorizontalOptions="CenterAndExpand" Padding="20" Margin="15"
                                  VerticalOptions="CenterAndExpand">
-                        <Label Text="{Binding Text}" HorizontalTextAlignment="Center" FontSize="20"
-                               FontAttributes="Bold" Style="{DynamicResource Ssp-Bold}" />
-                        <Label Text="{Binding Subtext}" HorizontalTextAlignment="Center" FontSize="16" Style="{DynamicResource Ssp-Regular}"/>
-                        <ProgressBar Progress="{Binding LoadingProgress}" Margin="20,0">
+                        <Label Text="{Binding Text}" HorizontalTextAlignment="Center" FontSize ="20"
```

Comment:
There's an unneeded space here in front of the `=`

Reply:
removed the extra space